docs/sources/tutorials/logs-and-relabeling-basics.md

```diff
@@ -259,7 +262,7 @@ echo 'level=warn msg="WARN: This is a warn level log!"' >> /tmp/alloy-logs/log.l
 echo 'level=debug msg="DEBUG: This is a debug level log!"' >> /tmp/alloy-logs/log.log
 ```
 
-Navigate to [localhost:3000/explore][] and switch the Datasource to `Loki`.
+Navigate to <localhost:3000/explore> and switch the Datasource to `Loki`.
```

Comment:
Do the angle brackets have a special meaning?

Reply:
Followup: The angle brackets around a URL is called autolinks in Git markdown. It renders as a link in the website output. The result is the same as the inline links or reference links. For consistency, I've switched it over to the more commonly used inline link markdown.  The result is the same in the rendered docs and it's not hard to do. :-)